app/models/site.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# The sites at which businesses have children in care
+class Site < ApplicationRecord
+  # Handles UUIDs breaking ActiveRecord's usual ".first" and ".last" behavior
+  self.implicit_order_column = 'created_at'
+
+  belongs_to :business
+  has_many :child_sites, dependent: :destroy
+  has_many :children, through: :child_sites
+
+  validates :active, inclusion: { in: [true, false] }
+  validates :name, presence: true, uniqueness: { scope: :business_id }
+  validates :address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zip, presence: true
+  validates :county, presence: true
+
```

Comment:
Should there be a validator for `qris_rating`?  I noticed that the Factory for it just creates values `1..5` -- so is that the valid range?

Reply:
I'm not sure about the business logic here; QRIS rating might be 1-5 in every state or it might be different; I think we want to prevent a validation because of the likelihood of us running across a state that does their QRIS ratings by letters or some other factor; I'll check with Chelsea though, good catch!

Reply:
<img width="711" alt="Screen Shot 2020-08-10 at 9 08 35 AM" src="https://user-images.githubusercontent.com/8364647/89791762-56664700-dae9-11ea-92ba-ac4794a1b9e8.png">
Yep, looks like they're all gonna be different 😭